CHANGELOG.md

```diff
@@ -454,4 +466,11 @@ thrown to caller to be dealt with so no need to log at this verbosity by pgjdbc
 [42.2.16]: https://github.com/pgjdbc/pgjdbc/compare/REL42.2.15...REL42.2.16
 [42.2.17]: https://github.com/pgjdbc/pgjdbc/compare/REL42.2.16...REL42.2.17
 [42.2.19]: https://github.com/pgjdbc/pgjdbc/compare/REL42.2.17...HEAD
-[Unreleased]: https://github.com/pgjdbc/pgjdbc/compare/REL42.2.19...HEAD
+## [Unreleased]
+### Changed
+
+### Added
+
+### Fixed
+
+[42.2.20]
```

Comment:
```suggestion
[42.2.17]: https://github.com/pgjdbc/pgjdbc/compare/REL42.2.16...REL42.2.17
[42.2.18]: https://github.com/pgjdbc/pgjdbc/compare/REL42.2.17...REL42.2.18
[42.2.19]: https://github.com/pgjdbc/pgjdbc/compare/REL42.2.18...REL42.2.19
[42.2.20]: https://github.com/pgjdbc/pgjdbc/compare/REL42.2.19...REL42.2.20
[Unreleased]: https://github.com/pgjdbc/pgjdbc/compare/REL42.2.20...HEAD
```

Reply:
@jorsol what is your comment ?

Reply:
It's a suggested change, in the `CHANGELOG.md` the last lines should not contain:
```
## [Unreleased]
### Changed

### Added

### Fixed

[42.2.20]
```
That's not correct, if they are added by the new tooling, then there is a problem.

The correct should be (for this release):
```
[42.2.18]: https://github.com/pgjdbc/pgjdbc/compare/REL42.2.17...REL42.2.18
[42.2.19]: https://github.com/pgjdbc/pgjdbc/compare/REL42.2.18...REL42.2.19
[42.2.20]: https://github.com/pgjdbc/pgjdbc/compare/REL42.2.19...REL42.2.20
[Unreleased]: https://github.com/pgjdbc/pgjdbc/compare/REL42.2.20...HEAD
```
Please note also that the link for 42.2.18 is missing.